ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -117,7 +117,12 @@ protected void init(Context context, AttributeSet attrs, int defStyleAttr, int d
                         res.getDisplayMetrics()));
         setTextSize(textSize);
 
-        animationDurationInMillis = DEFAULT_ANIMATION_DURATION;
+        if (arr.hasValue(R.styleable.TickerView_ticker_animationDuration)) {
+            //cannot use default here, as it upsets the compiler due to int vs long
```

Comment:
so this will just be `animationDurationInMillis = arr.getInt(R.styleable.TickerView_ticker_animationDuration, DEFAULT_ANIMATION_DURATION);`